owncloud/ios-app
# [feature/fastlane-app-group-identifier] Set App Group Identifier by Fastlane via Branding.plist

## Description
Set a custom app group identifier via Branding.plist this parameter. This value will be set by fastlane to all needed Info.plist keys. This is needed, if a customer is using an own resigning script which does not handle setting the app group identifier.

## Related Issue
#4918

## Motivation and Context


## How Has This Been Tested?
- add a key/value pair `build.app-group-identifier` to `Branding.plist` (or as advanced parameter in ownBrander)
- build app via fastlane (ownBrander)
- output IPA should contain the custom app group identifier in all `Info.plist` files

## Screenshots (if appropriate):

## Types of changes


## Checklist:
